std/format/write.d

```diff
@@ -1228,15 +1228,25 @@ Note:
 See_Also:
     $(LREF formattedWrite) which formats several values at once.
  */
-void formatValue(Writer, T, Char)(auto ref Writer w, auto ref T val, scope const ref FormatSpec!Char f)
+void formatValue(Writer, T, Char)(auto ref Writer w, auto ref T val, scope const ref FormatSpec!Char f) // TODO: make const(T) work
```

Comment:
Should we qualify `T` as `const` here? If I do I get a strange breakage in another Phobos module.

Reply:
Skipping this.